### DISCUSSION TITLE
Remote searches and tunnel speed

### DISCUSSION THREAD

**Discussion by hbiyik** [Tribler/tribler#7221] [Show and tell]
When download speed is suffering, a remote search to whatever increases the tunnel speed temporarily for an unknown reason.

https://www.youtube.com/watch?v=vfnsqaVjhqU

Check this one out. Open subtitles, and translate to english if you cant Russian.

Thats kind of funny.

**Comment by synctext** [comment]
Epic find!
A remote search also uses whopping amount of bandwidth. We are quite aggressive in trying to download the complete .torrent files, to ensure any download starts fast.

But if swarm speed inside download section gets a boost or unstuck, you're on to something...

**Comment by hbiyik** [comment]
I could reproduce this but not consistently, may be because my "ubuntu.iso" torrents do not have some many peers. Not so sure, but found it worth of sharing.
